### PR TITLE
Add test for find nodes zero distance

### PR DESCRIPTION
**Pull request by KolbyML**

### CODE REVIEW

File: simulators/portal-interop/src/main.rs [hunk 1]
```diff
@@ -401,3 +412,28 @@ dyn_async! {
...
+        let target_enr = match client_b.rpc.node_info().await {
+            Ok(node_info) => node_info.enr,
+            Err(err) => {
+                test.fatal(&format!("Error getting node info: {err:?}"));
+                return;
+            }
+        };
+
+        match client_a.rpc.find_nodes(target_enr.clone(), vec![0]).await {
+            Ok(response) => match response.enrs.get(0) {
```
Comment (kdeme) [inline simulators/portal-interop/src/main.rs:427]: Additionally we could test if `response.enrs.len() == 1`

**Review by kdeme — APPROVED** [review]

File: simulators/portal-interop/src/main.rs (hunk 1 above)
Comment (KolbyML) [reply simulators/portal-interop/src/main.rs]: resolved
